modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -16,7 +16,9 @@ namespace Godot.Collections
     /// interfacing with the engine. Otherwise prefer .NET collections
     /// such as <see cref="System.Array"/> or <see cref="List{T}"/>.
     /// </summary>
+#pragma warning disable CA1710
     public sealed class Array :
+#pragma warning restore CA1710
```

Comment:
This has been previously discussed in https://github.com/godotengine/godot/pull/65532#discussion_r967427306 where neikeq suggested to add it to `.editorconfig` directly.

I personally don't have a strong preference either way, but if we're going to keep the pragmas I like to always include the message in a comment so it's clear what the diagnostic is about without having to lookup the Id.

```suggestion
#pragma warning disable CA1710 // Identifiers should have correct suffix
    public sealed class Array :
#pragma warning restore CA1710
```

Reply:
I don't have a strong opinion either, I used `#pragma` here to be the less disruptive possible. But if most people agree, I'm also down to suppress it entirely from the `.editorconfig`.

Reply:
> I used `#pragma` here to be the less disruptive possible.

That was also what I was thinking in https://github.com/godotengine/godot/pull/65532. Let's keep the pragma and add the comment then, we can always change our minds in the future.